lola.html

```diff
@@ -0,0 +1,953 @@
+        </li>
+      </ol>
+
+      <p>
+        Existing servers seem to follow the “<b>leave them alone</b>” model rather than the <b>“rewrite”</b> model.  The way items propagate initially doesn’t have to define the way they are maintained.  In any case, trying to follow the rewrite model would sure involve some misses, failing back to the leave them alone model.
+      </p>
+
+      <p>
+        For ease of implementation, we lean towards assuming the “leave them alone” model.  Some of the reasons why:
+      </p>
+
+      <ul>
+        <li>How far should change propagation go? If Aurora moves her article and Cherry's Like gets updated does that propagate to Aurora’s subscribers? Do they get notified? </li>
+        <li>Where should change propagation go to?  Original follower list at the time the item was sent out? Aurora’s follower list today? 
+        </li>
+        <li>A rewrite model, if it were to be relied upon, would require many OTHER servers to participate in a data transfer specification, not just the two servers directly involved.  This increases the difficulty of adoption.
+        </li>
+        <li>
+          Since Activities/Likes can be disconnected in other ways, supporting breadcrumbs seems more useful than trying to achieve perfect rewriting.
+        </li>
+      </ul>
+
+      <p>
+        In addition to assuming that 3rd party servers leave their likes alone in many cases, this specification recommends that the destination server add breadcrumbs to Activity and Like objects to record their prior identities. This is beneficial in showing a nuanced view of older data.  If necessary, a server interested in validating a collection of “Likes” submitted to it, can mostly do so (except for accounts that have disappeared) if the object that the collection of “Likes” it is on maintains a history of past object IDs.  Note that this still ALLOWS other servers to fix up their Likes but they can do so immediately, later, or never.
+      </p>
+
+      <h3>Data Degradation</h3>
+        <p>Any account migration bears a risk of data loss or degradation.  A server cannot reasonably accept content it does not know how to store or present.  A roundtrip can result in data that is not in all ways, or even in important ways, identical to the data before it was migrated and migrated back.  This specification is intended to facilitate best-effort account migration rather than perfect duplication, with the philosophy that the perfect can be the enemy of the good.</p>
+
+        <p>Some consequences of this philosophy:</p>
+
+        <ul>
+          <li>Account migration might not be the best tool for users to do account backups.  It might be better than no tool, but trying to migrate data to another live location involves enough different choices than account backup, that it is likely that the result would be some data loss.</li>
```

Comment:
I like the idea of punting #14 to the export/import standard, because it's much easier there to achieve nuanced fallbacks for destination server not 100% understanding the imported content and offering the user choices about where to stick the things it doesn't understand at all and how to handle things it understands that have an extra property or two.  A PR making this punt explicit could close #14 imho, assuming the other interlocutors are on board with this "best-effort" philosophy